Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:10
+
+COPY . /etc/pangolin
+
+RUN cd /etc/pangolin \
+    && npm install \
+    && chmod +x ./bin/index.js \
+    && ln -s /etc/pangolin/bin/index.js /usr/local/bin/pangolin
+
+EXPOSE 10000
+EXPOSE 10001
+
+ENTRYPOINT ["pangolin"]
```

Suggestion for passing the door via parameters.

This way, it is possible to use the application inside the HEROKU where the port is different, or in any other host just changing the ENV.

Build with ```docker build --build-arg MY_SERVICE_PORT=80 -t image_tag```.
If the ENV is not informed it will use the default ports 10000 and 10001.

```suggestion
FROM node:10

ARG MY_SERVICE_PORT=10000
ARG MY_SERVICE_PORT_RPC=10001

COPY . /etc/pangolin

RUN cd /etc/pangolin \
    && npm install \
    && chmod +x ./bin/index.js \
    && ln -s /etc/pangolin/bin/index.js /usr/local/bin/pangolin

EXPOSE ${MY_SERVICE_PORT}
EXPOSE ${MY_SERVICE_PORT_RPC}

ENTRYPOINT ["pangolin"]
```